pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -448,30 +453,13 @@ func (b *BridgeBindMechanism) generateDhcpConfig() *cache.DhcpConfig {
 	return dhcpConfig
 }
 
-func (b *BridgeBindMechanism) getFakeBridgeIP() (string, error) {
-	ifaces := b.vmi.Spec.Domain.Devices.Interfaces
-	for i, iface := range ifaces {
-		if iface.Name == b.iface.Name {
+func getFakeBridgeIP(ifaceAlias string, libvirtDomainIfaces []v1.Interface) (string, error) {
```

Comment:
I'm continuing another comment that disappeared.
I've asked why `BridgeBindMechanism` was removed. You answered it is because it is in use by `PodNic`. Couldn't find where? Can you please share the link?